oeps/architectural-decisions/oep-0052/decisions/001-schema-representation.rst

```diff
@@ -0,0 +1,30 @@
+Schema Representation
```

Comment:
I'm not entirely clear on how to best represent the relationship between Avro and CloudEvent. Does this seem like the right ADR to help clarify?

Here is the [CloudEvent Avro specification example](https://github.com/cloudevents/spec/blob/v1.0.1/spec.avsc).

Here is a link to the [CloudEvent specification for Kafka bindings and binary encoding](https://github.com/cloudevents/spec/blob/v1.0.1/kafka-protocol-binding.md#32-binary-content-mode).